doc/source/reference/random/performance.rst

```diff
@@ -72,3 +72,88 @@ performance was computed using a geometric mean.
++---------------------+--------------+----------+----------+-----------+------------+----------+
+| 64-bit Unsigned Int |      5.51244 |  4.8276  |  6.01732 |   7.84796 |   10.0942  |  19.1023 |
++---------------------+--------------+----------+----------+-----------+------------+----------+
+| Uniform             |      4.88748 |  4.29088 |  3.14824 |   8.58    |   10.1495  |  19.1681 |
++---------------------+--------------+----------+----------+-----------+------------+----------+
+| Exponential         |      5.80104 |  5.77116 |  7.17204 |   9.68876 |   11.4676  |  20.4227 |
++---------------------+--------------+----------+----------+-----------+------------+----------+
+| Normal              |      8.18844 | 10.2346  | 12.0307  |  13.8725  |   14.8212  |  22.999  |
++---------------------+--------------+----------+----------+-----------+------------+----------+
+|**Performance Index**|        156.5 |    158.4 |    146.7 |       100 |       82.2 |     47.1 |
++---------------------+--------------+----------+----------+-----------+------------+----------+
+
+32-bit Windows
+~~~~~~~~~~~~~~
+
+The performance of 64-bit generators on 32-bit Windows is much lower than on 64-bit
+operating systems due to register width. DSFMT uses SSE2 when available, and so is less
+affected by the size of the operating system's register. MT19937, the generator that has been
+in NumPy since 2005, operates on 32-bit integers and so is close to DSFMT.
+
++---------------------+----------+-----------+--------------+----------+----------+------------+
+|                     |    DSFMT |   MT19937 |   Xoshiro256 |   Philox |    PCG64 |   ThreeFry |
++=====================+==========+===========+==============+==========+==========+============+
+| 32-bit Unsigned Int |  5.23144 |   5.80024 |      7.80116 |  20.6524 |  18.9328 |    19.8948 |
++---------------------+----------+-----------+--------------+----------+----------+------------+
+| 64-bit Unsigned Int |  8.68432 |   9.09488 |     12.5052  |  38.369  |  37.6123 |    40.0852 |
++---------------------+----------+-----------+--------------+----------+----------+------------+
+| Uniform             |  4.57136 |  11.292   |     14.3297  |  39.7076 |  42.3089 |    40.6009 |
++---------------------+----------+-----------+--------------+----------+----------+------------+
+| Exponential         | 12.7306  |  14.0114  |     15.6433  |  43.0479 |  43.6065 |    45.9283 |
++---------------------+----------+-----------+--------------+----------+----------+------------+
+| Normal              | 15.8798  |  17.0178  |     18.3056  |  45.317  |  46.8248 |    48.2998 |
++---------------------+----------+-----------+--------------+----------+----------+------------+
+|**Performance Index**|    127.6 |       100 |        81.3  |    29.7  |     29.7 |       28.8 |
++---------------------+----------+-----------+--------------+----------+----------+------------+
+
```

Comment:
My untrained eye would find it easier if you could keep the column order the same between the tables, even if it means the columns are not ranked by performance

Reply:
I would probably snip a few digits off the table. It isn't all that precise anyway.